index.html

```diff
@@ -61,7 +61,7 @@ <h3 class="letsconnect">LET'S CONNECT</h3>
       <section class="works-temp1 works1">
         <img src="images/SnapshootPortfolio(1).png" class="pic-temp1" alt="" />
```

Comment:
- [ ] Kindly add an appropriate alternative text to the `alt` attribute of all the project card images because the image is important for understanding the content of the project card. The image describes the project hence why it needs an `alt` text which is important for your browser accessibility and screen readers 🤓